optimize/backend/src/main/java/io/camunda/optimize/rest/security/cloud/CCSaaSSecurityConfigurerAdapter.java

```diff
@@ -105,6 +111,21 @@ public AuthenticationCookieFilter authenticationCookieFilter(final HttpSecurity
             .build());
   }
 
+  @Bean
+  @Order(-100)
```

Comment:
What is this annotation for? Could you add a comment explaining what `-100` is doing here? It's also used in the method itself

Reply:
I'm going to use a constant in both cases with value -100 to set them and I will also add comments.

@Order specifies the order of loading, the other specifies the order in the filter chain. I'm using a high priority value with both, so that this filter happens before anything. It's been working in my case, but if I am wrong with something, please feel free to let me know.

Reply:
Is the goal here to load this bean first? You can use `Ordered.HIGHEST_PRECEDENCE` instead of a magic number here